setup.cfg

```diff
@@ -0,0 +1,27 @@
+# coverage.py
+[coverage:run]
```

Comment:
You should be able to move all of these configurations into the `pyproject.toml` and remove the `setup.cfg` I think all bar maybe `flake8` (which I'd recommend replacing with `pylint`, this is on my backlog to do with AIP projects!) so you don't need multiple configuration files!

Reply:
May I ask, why replace flak8 with pylint?  Why not run both?

Reply:
Mostly because the goal it to remove the `setup.cfg` and `.flake8` config files and inline the configuration into `pyproject.toml`, with `flake8` until they resolve https://github.com/PyCQA/flake8/issues/234 this cannot be achieved 😒 

`pylint` does supporting reading from `pyproject.toml` though: https://pylint.pycqa.org/en/latest/user_guide/run.html#command-line-options

Reply:
I could do this in a later PR?

Reply:
Works for me. Will leave for you to resolve this thread with however you'd want to track this work.